package.json

```diff
@@ -13,16 +13,19 @@
 	"type": "module",
 	"exports": {
 		"types": "./index.d.ts",
-		"default": "./source/index.js"
+		"default": "./dist/index.js"
```

Comment:
I kinda want to go away from abbreviations.

Which one do you like the most?

- distribution
- output
- release
- production

Reply:
Maybe `build`? Of those, I like `output`.